### DISCUSSION TITLE
Add support for Polls

### DISCUSSION THREAD

**Discussion by brootle** [deso-protocol/dips#214] [Medium CIPs]
Polls is the essential feature for all social platforms that I know. Take Twitter or Facebook for example.

It's time to update DeSo backend and add support for Polls.

**Comment by brootle** [comment]
@diamondhands0 @maebeam @redpartyhat @AeonSw4n let's make this happen!🚀

**Reply by znmead** [reply]
At this point polls, in addition to many other uses, could be an essential tool for DAOs to implement as a governance mechanism in lieu of more advanced methods, and indeed could be built out with additional logic (smart service?) as a binding decision-making tool working towards full autonomy. 

Right now we have trusted agents. 

Fully support, thanks for bringing it up, Brootle.

**Comment by starykolarz** [comment]
Great minds think alike :-)

I thought about polls lacking from DeSo about 16 hours ago, 8 hours ago I implemented a first version basing on posts, so usable on all nodes.  Before we have it implemented in the protocol - check out @PollMe :-)

I fully support the idea of having in-protocol support!

**Comment by Lux1n** [comment]
Nice, thanks for building @pollme! gonna play around with it now
